### PR TITLE
Add ColorText enum and corresponding tests.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: src/Utility/ColorText.php [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5\Utility;
+
+/**
+ * Background colors.
```
Comment (samdark) [inline src/Utility/ColorText.php:8]:
```suggestion
 * Text colors.
```

**Review by samdark — APPROVED** [review]

File: src/Utility/ColorText.php (hunk 1 above)
Comment (terabytesoftw) [reply src/Utility/ColorText.php:8]: Name class `TextColor` or `ColorText` ?

Comment (terabytesoftw) [reply src/Utility/ColorText.php]: done.

File: src/Utility/TextColor.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5\Utility;
+
+/**
+ * Text colors.
+ *
+ * @see https://getbootstrap.com/docs/5.3/utilities/colors/#colors
+ */
+enum TextColor: string
```
Comment (vjik) [inline src/Utility/TextColor.php:12]: `text-black-50` and `text-white-50` are missing

**Review by vjik — CHANGES_REQUESTED** [review]

File: src/Utility/TextColor.php (hunk 2 above)
Comment (terabytesoftw) [reply src/Utility/TextColor.php:12]: image

**Review by vjik — APPROVED** [review]

**Pull request merged**